composer.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "lcobucci/jwt-rsassa-pss",
+    "description": "An extension to lcobucci/jwt to handle RSASSA-PSS algorithms",
+    "license": [
+        "MIT"
+    ],
+    "type": "library",
+    "keywords": [
+        "JWT"
+    ],
+    "require": {
+        "php": "~8.2.0 || ~8.3.0 || ~8.4.0",
+        "lcobucci/jwt": "^5.4.2",
+        "phpseclib/phpseclib": "^3.0.42"
+    },
+    "require-dev": {
+        "infection/infection": "^0.29.8",
+        "lcobucci/coding-standard": "^11.1",
+        "phpstan/extension-installer": "^1.4.3",
+        "phpstan/phpstan": "^2.0.3",
+        "phpstan/phpstan-deprecation-rules": "^2.0.1",
+        "phpstan/phpstan-phpunit": "^2.0.1",
+        "phpstan/phpstan-strict-rules": "^2",
+        "phpunit/phpunit": "^11.4.4"
+    },
+    "autoload": {
+        "psr-4": {
+            "Lcobucci\\JWTRsaPss\\": "src"
```

Comment:
Cunningham's Law

Reply:
I'd follow the namespace of the main library, which gives more familiarity and (in case this becomes a successful thing) allows us to more easily integrate it back.

Reply:
`PSR-4` has a conflict with the abstract class here, I chose the easier `classmap` which should be fine here